markdown-scheme_v3.Rmd

```diff
@@ -0,0 +1,393 @@
+    \item Geographic areas: Format: Name, Code, Type
+    \begin{lstlisting}
+      Lebanon, LBN, ISO-3166 Alpha-3
+    \end{lstlisting}
+    \item Keywords: No keywords if none are given; Yes if given.
+    \item JEL codes:
+      \begin{verbatim}
+        ID D63
+        Name Equity, Justice, Inequality
+        Parent ID (empty)
+        Vocabulary JEL Classifications
+        URL https://www.aeaweb.org/econlit/jelCodes.php?view=jel
+      \end{verbatim}
+\end{enumerate}
+
+## Processes and Output
+The following fields exist
+\begin{enumerate}
+    \item Process: empty for PP, tbd for WP
+    \item Institutional review board: empty
+    \item Approval process: empty
+    \item Reviews and comments:
+      \begin{itemize}
+        \item WP: If available (tbd, 2023-06-22)
+        \item PP: empty
+      \end{itemize}
+    \item Output: 
+        \begin{itemize}
+          \item Type    
+            \begin{itemize}
+              \item WP: Working Paper
+              \item PP: Published Paper (not paper, not article, not Article)
+            \end{itemize}
+          \item Title: Title of the paper
+          \item Authors: Authors: all comma separated; first name, last name. Patty Smith, Patty Smith, Petty Smith
+          \item Description: \textcolor{red}{Add kind and No of external working paper if applicable}
+            \begin{lstlisting}[breaklines]
+                Toulouse School of Economics Working Papers No 1391
+            \end{lstlisting}
+          \item DOI: YES (if it exists)
+          \item URL: YES (Editor displays an error at https, but it is accepted)
+        \end{itemize}
+    \item Language: English, EN
+\end{enumerate}
+
+## Data
+Sufficient are
+\begin{enumerate}
+    \item Data statement:\footnote{ Description: An overall statement on the data used in the project. A spearate field is provided to list and document the origin and key characteristics of the datasets}. For external packages, the following rule:
+        \begin{itemize}
+          \item If all data is public:
+            \begin{itemize}
+            \item Fill in: "All data is public and contained in the (external)         reproducibility package."
+            \item Add data availability statement if available from journal package; often separate from the zip, has to be copied over manually.
+            \end{itemize}
+          \item If some data is confidential: Paste from the readme notes about confidentiality.
+        \end{itemize}
+    \item Datasets:\footnote{Description of this field: "This field is used to provide an itemized list of datasets used in the project. The data are not documented here (specific metadata are available for documenting data of different types, like the DDO for microdata, the ISO 19139 for geographic datasets etc)"} Not used for PP, will be used for WP, if they data sets are of proper value and stored in another WB repository (like microdata library). \textcolor{red}{No such case arose so far. TBD}
```

Comment:
certainly applies for PPs. i would think more common for PPs to have data published separately and called from rep package (e.g., data are catalogued elsewhere and referenced in the data statement). for WPs we may be directly involved in cataloguing so that is different.

Reply:
I will change the wording